content/blog/2024/01/kaoto-release-2.0.0/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Kaoto v2.0 release"
+date: 2024-06-06
+draft: false
+authors: [lordrip]
+categories: ["Releases", "Tooling", "Kaoto"]
+preview: "Kaoto 2.0 has been released"
+---
+
+<p align="center">
+  <img src="CamelKaoto.png" alt="Kaoto" height="400"/>
```

Comment:
Where is that image?

Reply:
It was missing from the images batch, I found it and I'll upload it